frontier_navigation:
    ros__parameters:
      global_frame: "map"
      base_frame: "base_link"
      map_topic: "map"
      exploration_rate: 5.0
      min_aborted_goals_distance: 0.2
      min_frontier_size: 20
      search_only_free_space: true
      frontier_size_scaling_factor: 0.0 # TODO: tune this
